pixi.toml

```diff
@@ -156,5 +162,22 @@ docs = { features=["docs"] }
 [tasks]
 # builds in the obelisk_ws directory only if src has changes
 source_rosbase = { cmd="bash source_rosbase.sh", cwd="scripts", env={ NOOB="true" } }
-build = { cmd="colcon build --symlink-install --parallel-workers $(nproc)", cwd="obelisk_ws", inputs=["src"], depends-on=["source_rosbase"] }
+ros-build = { cmd="colcon build --symlink-install --parallel-workers $(nproc)", cwd="obelisk_ws", inputs=["src"], depends-on=["source_rosbase"] }
```

Comment:
we need to pick a consistent standard for tasks: either underscores or hyphens. i personally prefer underscores here, but you can make a case the other way.

Reply:
I think hyphens are more standard when interfacing with the CLI. See [here](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html). I think the hyphens are also a bit clearer.

Lmk what you think. I can change the others to hyphens in this PR.

Reply:
Ah I see what you mean. Yes, that makes sense. Can you do a quick scan for the underscores I have for CLI commands and turn them all into hyphens?